### PR TITLE
Bundle whitelist

### PR DESCRIPTION
**Pull request by lenuhc**
* Added configuration for adding bundles to a whitelist that will be ignored when running the Bundle Health-Check

### CODE REVIEW

File: bundle/src/main/java/com/shinesolutions/healthcheck/hc/impl/ActiveBundleHealthCheck.java [hunk 1]
```diff
@@ -28,9 +32,14 @@
     private static final String BUNDLE_ACTIVATION_POLICY = "Bundle-ActivationPolicy";
     private static final String LAZY_ACTIVATION_POLICY   = "lazy";
 
+    @Property(label = "Ignored Bundles", description = "The bundles that will be ignored in the Active Bundle Health-Check")
+    protected static final String IGNORED_BUNDLES = "bundles.ignored";
+    protected static String[] ignoredBundles;
```
Comment (mickleroy) [inline bundle/src/main/java/com/shinesolutions/healthcheck/hc/impl/ActiveBundleHealthCheck.java]: shouldn't be `static`

File: bundle/src/main/java/com/shinesolutions/healthcheck/hc/impl/ActiveBundleHealthCheck.java [hunk 2]
```diff
@@ -28,9 +32,14 @@
...
+
     @Activate
     protected void activate(ComponentContext context) {
         bundleContext = context.getBundleContext();
+        ignoredBundles = PropertiesUtil.toStringArray(context.getProperties().get(IGNORED_BUNDLES));
```
Comment (mickleroy) [inline bundle/src/main/java/com/shinesolutions/healthcheck/hc/impl/ActiveBundleHealthCheck.java]: can you make sure this works when no config file is provided

File: bundle/src/test/java/com/shinesolutions/healthcheck/hc/impl/ActiveBundleHealthCheckTest.java [hunk 3]
```diff
@@ -79,6 +86,28 @@ public void testLazyActivationBundle() {
     }
 
     @Test
+    public void testIgnoredBundles() {
```
Comment (mickleroy) [inline bundle/src/test/java/com/shinesolutions/healthcheck/hc/impl/ActiveBundleHealthCheckTest.java]: should add a test for when `properties` is null.
